### ISSUE TITLE
Accessibility:List: Extra incorrect information “read only” is conveyed when list items and interactive list items are accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3459]
Description: Extra incorrect information “read only” is conveyed when list items and interactive list items are accessed. 
Clickable information about interactive list items is not conveyed to the user. 

Expected: Such incorrect extra information should not be conveyed. Only relevant information should be read out when each & every UI is accessed

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94524236-f9b51c00-024f-11eb-9c6e-9c199f07fadd.png)

![image](https://user-images.githubusercontent.com/32538291/94524246-fe79d000-024f-11eb-8f98-5ebf14c0507b.png)

![image](https://user-images.githubusercontent.com/32538291/94524260-02a5ed80-0250-11eb-9b44-1aa62268a027.png)

**Comment by sKudum** [comment]
No reproducible is latest release.hence closing it.
